deltachat-ios/Controller/ContactDetailViewController.swift

```diff
@@ -95,6 +95,20 @@ class ContactDetailViewController: UITableViewController {
         return cell
     }()
 
+    private lazy var locationsCell: UITableViewCell = {
+        let cell = UITableViewCell(style: .value1, reuseIdentifier: nil)
```

Comment:
I'm so used to using `UIListContentConfiguration` these days :D

Reply:
`UIListContentConfiguration` is what renders the settings eg. in the iOS system settings? anyways, requires iOS14, so not today :)